lib/internal/http2/core.js

```diff
@@ -1055,9 +1063,9 @@ function createClientSession(options, socket) {
   socket.on('pause', socketOnPause);
   socket.on('drain', socketOnDrain);
 
-  session.on('error', sessionOnError);
-  session.on('stream', sessionOnStream);
-  session.on('selectPadding', sessionOnSelectPadding);
+  // session.on('error', sessionOnError);
+  // session.on('stream', sessionOnStream);
+  // session.on('selectPadding', sessionOnSelectPadding);
```

Comment:
These are event handlers for the server. They each use `session[kServer]` which is undefined for clients. I was wondering if we need these handlers at all, or if they should do something else perhaps. Thoughts?